### DISCUSSION TITLE
CPEDITOR not closing.

### DISCUSSION THREAD

**Discussion by baekje-tls** [cpeditor/cpeditor#1164] [Q&A]
I had downloaded cpeditor, then I put it in full-screen.
I had received the following message:
<img width="436" alt="Screenshot 2023-04-04 at 8 10 50 PM" src="https://user-images.githubusercontent.com/128426158/229963161-cb7153eb-9db1-4ffd-bf6e-6e8482352300.png">
Normal, right?
However, when I clicked OK or when I pressed command w and command q, the window wouldn't close. I tried restarting and shutting down my computer, but I receive this message:
<img width="627" alt="Screenshot 2023-04-04 at 8 09 08 PM" src="https://user-images.githubusercontent.com/128426158/229963129-311cdc8a-38bb-485b-852c-8273d834e6e4.png">
What should I do?
I am no longer able to use my computer as normal.

**Comment by coder3101** [comment]
Does #1166 fixes this for macOS too @ouuan ?

**Reply by ouuan** [reply]
I'm not sure if #1166 actually fixes any issues that are encountered by anyone. It's just more robust in theory.
